sorbet/rbi/gems/ansi@1.5.0.rbi

```diff
@@ -0,0 +1,281 @@
+# This file is autogenerated. Do not edit it by hand. Regenerate it with:
+#   tapioca generate ansi
```

Comment:
After adding `require ansi/code` into sorbet/tapioca/require.rb I didn't want to remove all RBIs, run `bundle exec tapioca sync` and only commit the ansi RBI.

Instead I regenerated ansi only. That's why it wasn't generated by `sync`